app/src/main/java/lt/vilnius/tvarkau/fragments/NewReportFragment.kt

```diff
@@ -440,38 +439,51 @@ class NewReportFragment : BaseFragment(),
                     val geocoder = Geocoder(context)
                     val place = PlacePicker.getPlace(context, data)
 
-                    var addresses = listOf<Address>()
+                    val addresses = mutableListOf<Address>()
```

Comment:
`geocoder.getFromLocation` might return a list of values, a `null` or throw exception.

It's way easier down the line to check if list is empty rather than checking if list is null or empty.